fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/cache/FHIRPersistenceJDBCTenantCache.java

```diff
@@ -33,14 +33,14 @@
      */
     public FHIRPersistenceJDBCCache getCacheForTenantAndDatasource() {
         final String cacheKey = getCacheKeyForTenantDatasource();
-        
+
         // Obtain the cache for the current tenant/datasource, creating a new one if need be
         FHIRPersistenceJDBCCache result = cacheMap.computeIfAbsent(cacheKey, (k) -> createCache(k));
         if (result == null) {
             // exception handled inside createCache (which will have been logged)
-            throw new IllegalStateException("createCache failed");
+            throw new IllegalStateException("createCache failed " + cacheKey);
```

Comment:
Worth noting it is intentional that this message is included in the request response?

Reply:
Yes, that's right I'll add a comment

Reply:
comment is added